### ISSUE TITLE
Uworld_qid_to_anki_search add on is no more working

### ISSUE THREAD

**Issue by Veggiecupcakes** [AnKing-VIP/ankiweb-placeholder#106]
Uworld_qid_to_anki_search add on

So I had subscribed to a bronze membership with AnkingVIP and got a couple add-ons including the uworld qid search add on, it is now not working any more with the display showing that there are matching cards to question ids but nothing on the browser screen. 
My subscription with AnkingVIP ended earlier this month. 
![image](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/170955911/28c8886b-822b-440c-b252-3777287fc76c)
![image](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/170955911/37f9df74-0473-4498-a30b-539070151462)
![image](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/170955911/9bbb0d47-bf65-41fe-9d58-a54fa2708540)

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [x] I've restarted Anki to see if it helps
 - [x] I've verified that I use the latest version of the add-on by redownloading it from AnkiWeb
 - [x] I've tried to disable other add-ons to see if there are any interactions present
 - [ ] My issue disappears when I hold shift while starting Anki.
 - [x] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

I am on Anki ⁨2.1.64 Qt 6

**Comment by AnKingMed** [comment]
Do you have the latest version of V12?

**Comment by Veggiecupcakes** [comment]
Yes I do, I am signed to AnkiHub and I sync regularly. 
Also here is my add on config
![image](https://github.com/AnKing-VIP/ankiweb-placeholder/assets/170955911/5145b6a6-c872-45d5-bb78-8329b232c0a0)

**Comment by AnKingMed** [comment]
Can you try changing test to "Auto" and see if that works?

**Comment by Veggiecupcakes** [comment]
Oh it just worked!! Thank you so much.

**Comment by Veggiecupcakes** [comment]
I am also surprised bc I just did that a while go and it didnt work hahah

**Comment by AinaRattu** [comment]
> Can you try changing test to "Auto" and see if that works?

Hi there, I recently downloaded this and I'm having the same problem. Can you clarify what you mean by changing test to auto?

**Comment by AnKingMed** [comment]
When you click "UWorld" in the top menu bar (in the browser view), there is an option to select test

**Comment by AinaRattu** [comment]
> When you click "UWorld" in the top menu bar (in the browser view), there is an option to select test

Hi there, did this. Now only 1-2 cards will show up. I have tried all settings from this and I get an error with downloading the add-on. What should I do next?

**Comment by AnKingMed** [comment]
Can you send a photo of your settings?

Are you using v12 on AnkiHub?

**Comment by AinaRattu** [comment]
![Image](https://github.com/user-attachments/assets/9eaf6fae-f1b1-4bdc-aeb3-43d7e913d777)

I am using v12 on AnkiHub, have done what was typed out above with "Auto" test.

**Comment by AnKingMed** [comment]
This is the screen I wanted. Double check yours looks like this.

Can you send me some QIDs that aren't working?

![Image](https://github.com/user-attachments/assets/d31031db-708b-4300-a11f-b5f6bd952541)

**Comment by AinaRattu** [comment]
> This is the screen I wanted. Double check yours looks like this.
> 
> Can you send me some QIDs that aren't working?
> 
> ![Image](https://github.com/user-attachments/assets/d31031db-708b-4300-a11f-b5f6bd952541)

![Image](https://github.com/user-attachments/assets/3446f22f-4999-4c96-8d3f-ddc4067865f6)

![Image](https://github.com/user-attachments/assets/5c7990b9-f7c6-4c24-a84c-b769010e2c51)

**Comment by AnKingMed** [comment]
Is your deck up to date? I was able to find cards with those tags. For example nid:1473375621493

**Comment by AinaRattu** [comment]
I thought it was; it should be V12. Any other suggestions?

**Comment by AnKingMed** [comment]
V12 is on AnkiHub. You need to sync regularly to get updates. Have you synced recently?

**Comment by AinaRattu** [comment]
Yes I have. I have the Anking-VIP $5 membership. Do I need to buy the Ankihub membership to have the Step 2 tags?
________________________________

**Comment by AnKingMed** [comment]
AnKing-VIP is separate. 

AnkiHub is here: https://www.ankihub.net/step-deck

**Comment by AinaRattu** [comment]
So I need to pay for the Anki Hub membership to receive the Step 2 tags?
________________________________

**Comment by AnKingMed** [comment]
yes. And they receive frequent updates

**Comment by AinaRattu** [comment]
Ok got it. Thank you for your help!
________________________________
